docs/en/stack/ml/nlp/ml-nlp-autoscaling.asciidoc

```diff
@@ -0,0 +1,144 @@
+[[ml-nlp-auto-scale]]
+= Trained model autoscaling
```

Comment:
Who coined this term "trained model autoscaling". Haven't seen that one before.

I think I've lost track of how many names this features has...